app/governance/proposal/page.tsx

```diff
@@ -162,37 +208,102 @@ export default function Page() {
           <Text opacity={0.4}>{proposal.description}</Text>
         </div>
       </div>
+      <div className={styles.proposalInfoContainer}>
+        <div className={styles.graphAndVoteContainer}>
+          {isActive && (
+            <div className={styles.proposalCardContainer1}>
+              <div className={styles.detailsHeader}>
+                <Text font="proto_mono">Select an option to vote</Text>
+              </div>
+              <div
+                className={styles.votingBox}
+                style={
+                  isActive
+                    ? {
+                        height: "100%",
+                        padding: "10px 0px 0px 0px",
+                      }
+                    : {
+                        height: "100%",
+                        padding: "0px 0px 20px 0px",
+                      }
+                }
+              >
+                <div className={styles.proposalInfoRow1}>
+                  <VoteBox option={VoteOption.YES} idx={0} />
```

Comment:
wouldn't the option being enough to figure out what's colors to apply? index seems a bit unnecessary to achieve that goal.

Reply:
This refactor was done by David. I just realised it's not done by me

Reply:
Changed this anyway in the latest push